superset-frontend/plugins/plugin-chart-echarts/test/utils/series.test.ts

```diff
@@ -628,226 +630,231 @@ describe('formatSeriesName', () => {
     );
   });
 
-  describe('getLegendProps', () => {
-    it('should return the correct props for scroll type with top orientation without zoom', () => {
-      expect(
-        getLegendProps(
-          LegendType.Scroll,
-          LegendOrientation.Top,
-          true,
-          theme,
-          false,
-        ),
-      ).toEqual({
-        show: true,
-        top: 0,
-        right: 0,
-        orient: 'horizontal',
-        type: 'scroll',
-        ...expectedThemeProps,
-      });
+  it('should normalize non-UTC string based timestamp', () => {
+    const annualTimeFormatter = getTimeFormatter('%Y');
+    expect(
+      formatSeriesName('1995-01-01 00:00:00.000000', {
+        timeFormatter: annualTimeFormatter,
+        coltype: GenericDataType.TEMPORAL,
+      }),
+    ).toEqual('1995');
+  });
+});
+
```

Comment:
bycatch: the `getLegendProps` block was incorrectly nested inside the `formatSeriesName` block - check with "Hide whitespace" to see the relevant changes.